storybook/index.html

```diff
@@ -268,6 +268,19 @@ <h1 class="sectionHeading">Hello</h1>
                     </div>
                 </section>
 
+                <section class="demo" data-module-name="subFooter">
+                        <div class="subFooter">
+                            <a class="subFooter--socials" href="#"><img class="subFooter--svg" src="../images/footerMail.svg"></a>
+                            <a class="subFooter--socials" href="#"><img class="subFooter--svg" src="../images/instagram.svg"></a>
+                            <a class="subFooter--socials" href="#"><img class="subFooter--svg" src="../images/twitter.svg"></a>
+                            <a class="subFooter--socials" href="#"><img class="subFooter--svg" src="../images/facebook.svg"></a>
+                            <a class="subFooter--socials" href="#"><img class="subFooter--svg" src="../images/youtube.svg"></a>
+                            <a class="subFooter--socials" href="#"><img class="subFooter--svg" src="../images/footerLinkedin.svg"></a>
+                            <a class="subFooter--socials" href="#"><img class="subFooter--svg" src="../images/yelp.svg"></a>
+                            <a class="subFooter--text"><p class="subFooter--copyright">© 2020 BAY AREA MURAL PROGRAM</p></a>
```

Comment:
here we don't really want an `<a>` element for the copyright notice (because it doesn't navigate anywhere).

also I would just merge the `subFooter--copyright` and `subFooter--text` classes into one... and "copyright" is probably the better name to keep, because it's more specific.

you may also find that, depending on the tag that you use, you don't need the `margin: 0`. for instance maybe a `<span>` would just do the right thing already.

Reply:
e.g. HTML

```html
<p class="subFooter--copyright">© 2020 BAY AREA MURAL PROGRAM</p>
```

```css
.subFooter--copyright {
    text-align: right;
    flex-grow: 2;
    color: #DDDDDD;
    white-space: nowrap;
    margin: 0;
}
```